sycl/include/CL/sycl/detail/cg.hpp

```diff
@@ -62,7 +110,9 @@ class CG {
     FILL_USM = 11,
     PREFETCH_USM = 12,
     CODEPLAY_INTEROP_TASK = 13,
-    CODEPLAY_HOST_TASK = 14
+    CODEPLAY_HOST_TASK = 14,
+    KERNEL_V2 =
+        getVersionedCGType(KERNEL, static_cast<unsigned int>(CG_VERSION::V1)),
```

Comment:
Nit: matching two versions would be more intuitive
```suggestion
    KERNEL_V1 =
        getVersionedCGType(KERNEL, static_cast<unsigned int>(CG_VERSION::V1)),
```